src/coreComponents/physicsSolvers/multiphysics/MultiphasePoromechanicsSolver.cpp

```diff
@@ -128,19 +135,36 @@ void MultiphasePoromechanicsSolver::assembleSystem( real64 const GEOSX_UNUSED_PA
 
     string const flowDofKey = dofManager.getKey( CompositionalMultiphaseBase::viewKeyStruct::elemDofFieldString() );
 
-    poromechanicsMaxForce =
-      assemblyLaunch< constitutive::PorousSolidBase,
-                      poromechanicsKernels::MultiphasePoromechanicsKernelFactory >( mesh,
-                                                                                    dofManager,
-                                                                                    regionNames,
-                                                                                    viewKeyStruct::porousMaterialNamesString(),
-                                                                                    localMatrix,
-                                                                                    localRhs,
-                                                                                    flowDofKey,
-                                                                                    flowSolver()->numFluidComponents(),
-                                                                                    flowSolver()->numFluidPhases(),
-                                                                                    FlowSolverBase::viewKeyStruct::fluidNamesString() );
-
+    if( m_isThermal )
+    {
+      poromechanicsMaxForce =
+        assemblyLaunch< constitutive::PorousSolidBase,
```

Comment:
if you are struggling with compilation times on Travis. I have created a "dispatch" to only used `PorousSolid<ElasticIsotropic>`. It's obviously just a temporary fix.

Reply:
Thanks, that is very useful. I used the reduced dispatch for both single-phase THM and multiphase THM, so this PR should actually reduce the compilation time for everyone else now.